scipy/_lib/tests/test__util.py

```diff
@@ -347,17 +345,13 @@ def test_contains_nan_with_strings(self):
         data4 = np.array([["1", 2], [3, np.nan]], dtype='object')
         assert _contains_nan(data4)[0]
 
-    @skip_xp_backends('jax.numpy',
-                      reason="JAX arrays do not support item assignment")
-    @pytest.mark.usefixtures("skip_xp_backends")
     @array_api_compatible
     @pytest.mark.parametrize("nan_policy", ['propagate', 'omit', 'raise'])
     def test_array_api(self, xp, nan_policy):
         rng = np.random.default_rng(932347235892482)
         x0 = rng.random(size=(2, 3, 4))
         x = xp.asarray(x0)
-        x_nan = xp_copy(x, xp=xp)
-        x_nan[1, 2, 1] = np.nan
+        x_nan = xp_at(x)[1, 2, 1].set(np.nan)
```

Comment:
cosmetic: what do you think about following the pattern used elsewhere and calling this like

```python
import array_api_extra as xpx
...
xpx.at(...)
```

I think that nicely conveys that this is directly from array_api_extra rather than our own custom helper